apiserver/facades/controller/migrationtarget/migrationtarget_test.go

```diff
@@ -334,77 +334,6 @@ func (s *Suite) TestAdoptCAASResources(c *gc.C) {
 	broker.Stub.CheckCall(c, 0, "AdoptResources", s.callContext, st.ControllerUUID(), version.MustParse("3.2.1"))
 }
 
-func (s *Suite) TestCheckMachinesInstancesMissing(c *gc.C) {
```

Comment:
These are still needed for IAAS models aren't they?

Reply:
The logic that these tests are testing has been refactored to common/credentialcommon package in one of the previous PRs. These tests has been moved there as well.